app/Providers/RepositoryServiceProvider.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Providers;
+
+use Illuminate\Support\ServiceProvider;
+
+class RepositoryServiceProvider extends ServiceProvider
+{
+    protected static $repositories = [
+//        [
+////            Repo of Confessions
+//            'App\Repositories\Contracts\ConfessionRepository',
+//            'App\Repositories\Eloquents\ConfessionEloquentRepository',
+//        ],
+        [
+//            Repo of Topics
+            'App\Repositories\Contracts\TopicRepository',
+            'App\Repositories\Eloquents\TopicEloquentRepository',
+        ],
+        [
+//          Repo of Posts
+            'App\Repositories\Contracts\PostRepository',
+            'App\Repositories\Eloquents\PostEloquentRepository',
+        ],
+//        [
+////          Repo of Reports
+//            'App\Repositories\Contracts\ReportRepository',
+//            'App\Repositories\Eloquents\ReportEloquentRepository',
+//        ],
+    ];
+
+    /**
+     * Bootstrap services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        //
+    }
+
+    /**
+     * Register services.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        foreach (static::$repositories as $repository) {
+            $this->app->bind(
+                $repository[0],
+                $repository[1]
+            );
+        }
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Files.EndFileNewline.NoneFound</br>**Severity**: error</br>**File**: app/Providers/RepositoryServiceProvider.php</br>**Line**: 56</br>Expected 1 newline at end of file; 0 found